src/sentry/integrations/vercel/webhook.py

```diff
@@ -255,7 +260,10 @@ def _delete(self, external_id, configuration_id, request):
         #      organization integration AND the integration (since there is only one)
         #  2.) we already deleted the organization integration tied to this configuration
         #      and the remaining one is for a different org (and configuration)
-        if len(orgs) == 1:
+        if (
+            len(orgs) == 1
+            and features.has("organizations:vercel-integration-webhooks", orgs[0]) == new_webhook
```

Comment:
I just want to confirm that this is the expected logic here:
 - feature enabled: we end up in this branch when new_webhook is true, and not when it's false
 - feature disabled: we end up in this branch when new_webhook is False, and not when True